TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -346,8 +354,23 @@ public static void main(String[] args)
                      Duration.ofSeconds(params.lookaheadTimeSeconds));
              var authTransformer = buildAuthTransformerFactory(params))
         {
-            var tr = new TrafficReplayer(uri, params.transformerConfig, authTransformer,
-                    params.allowInsecureConnections, params.numClientThreads,  params.maxConcurrentRequests);
+            if (params.transformerConfigFile != null && !params.transformerConfigFile.isBlank() && params.transformerConfig != null && !params.transformerConfig.isBlank()) {
+                System.err.println("Specify either --transformer-config or --transformer-config-file, not both.");
+                System.exit(4);
+            }
+            String configContent = null;
+            if (params.transformerConfigFile != null && !params.transformerConfigFile.isBlank()) {
+                try {
+                    configContent = Files.readString(Paths.get(params.transformerConfigFile), StandardCharsets.UTF_8);
+                } catch (IOException e) {
+                    System.err.println("Error reading transformer configuration file: " + e.getMessage());
+                    System.exit(5);
+                }
+            } else if (params.transformerConfig != null && !params.transformerConfig.isBlank()) {
+                configContent = params.transformerConfig;
+            }
```

Comment:
Please put these lines into a helper function to return the config (or null).  The helper function could throw or call exit() directly.  We'll do a rewrite of all of this parsing logic that's getting spread out in the near future.